ipfx/dataset/ephys_nwb_data.py

```diff
@@ -63,13 +64,17 @@ def __init__(self,
                  validate_stim: bool = True,
                  ):
 
-        super().init(ontology=ontology, validate_stim=validate_stim)
+        super().__init__(ontology=ontology)
+        self.nwb_file = nwb_file
+        
         if load_into_memory:
             with open(nwb_file, 'rb') as fh:
-                nwb_file = BytesIO(fh.read())
+                data = BytesIO(fh.read())
 
-        self.nwb_file = nwb_file
-        self.nwb = NWBHDF5IO(nwb_file, mode='r').read()
+            _h5_file = h5py.File(data, "r")
```

Comment:
might want to hang onto this so methods that rely on h5 access don't have to reach through the nwbfile